.github/workflows/release.yml

```diff
@@ -97,7 +95,7 @@ jobs:
           tag: ${{ steps.tag_version.outputs.new_tag }}
           name: ${{ steps.tag_version.outputs.new_tag }}
           bodyFile: ${{ steps.release_notes.outputs.release_notes }}
-          artifacts: "dist.zip,desktop.zip"
+          artifacts: "desktop.zip"
```

Comment:
Maybe add the version number to the file so that after downloading you'd know which version it is?

Reply:
Done